src/pages/Index.vue

```diff
@@ -586,7 +599,7 @@
 		}
 
 		.heroMain {
-			display: flex;
+			/*display: flex;*/
```

Comment:
Please do not add commented property if there are no explanation about why it's commented.
Either uncomment it, or remove it!